insights/parsers/luksmeta.py

```diff
@@ -0,0 +1,79 @@
+"""
+luksmeta - command ``luksmeta show``
+====================================
+This class provides parsing for the output of luksmeta <device_name>.
```

Comment:
it should be `luksmeta -d <device_name>` here in the doc?

Reply:
You may miss this one.

Reply:
Fixed now